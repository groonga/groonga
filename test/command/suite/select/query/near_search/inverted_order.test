table_create Memos TABLE_PAT_KEY ShortText
column_create Memos content COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizer NormalizerAuto
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content

load --table Memos
[
{"_key":"alphabets1", "content": "a b c"}
]

select --table Memos --match_columns content --query '*N"c b a"' --output_columns _score,_key
