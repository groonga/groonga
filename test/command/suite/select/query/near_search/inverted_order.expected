table_create Memos TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Memos content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content
[[0,0.0,0.0],true]
load --table Memos
[
{"_key":"alphabets1", "content": "a b c"}
]
[[0,0.0,0.0],1]
select --table Memos --match_columns content --query '*N"c b a"' --output_columns _score,_key
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_score",
          "Int32"
        ],
        [
          "_key",
          "ShortText"
        ]
      ],
      [
        1,
        "alphabets1"
      ]
    ]
  ]
]
